mautic/mautic
# Emails ctr calculation

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | N/A
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A




#### Description:

Updated "Stats" column of Email table, added two items:
* clicks ( number of total clicks of links in email )
* CTR ( Click-through rate of links inside email)

These stats are included also single email view, where they reference to single link in "Click counts" table.

![ctrcalc](https://user-images.githubusercontent.com/50916237/109320164-01eda380-7850-11eb-9d6f-f9aebaf2e373.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create an email with link inside
3. Inside emal try to click link
4. Check new values inside emails table and click counts table in email view




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9689"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
